File: P120923.py
def solution(num, total):
    answer = []
    for i in range(-1000, 1001):
        tmp = 0
        for j in range(i, i+num):
            tmp += j
        if tmp == total:
            for j in range(i, i+num):
                answer.append(j)
            break
    return answer

File: P120923.java
import java.util.*;

class P120923 {
    public int[] solution(int num, int total) {
        ArrayList<Integer> arr = new ArrayList<>();
        for (int i=-1000; i<=1000; i++) {
            int tmp = 0;
            for (int j=i; j<i+num; j++) {
                tmp += j;
            }
            if (tmp == total) {
                for (int j=i; j<i+num; j++) arr.add(j);
                break;
            }
        }
        int[] answer = new int[arr.size()];
        for (int i=0; i<answer.length; i++) answer[i] = arr.get(i);
        return answer;
    }
}